main.go

```diff
@@ -484,3 +542,8 @@ func setupGUI(privateKey *ecdsa.PrivateKey, messenger *client.Messenger) error {
 
 	return nil
 }
+
+// @todo move?
+func Calc(count uint64, time int64) int64 {
```

Comment:
probably `mvds.Calc` or something like that... though this function definitely needs more explanation and probably a better naming...